New results type of grouped annotations elan files at [ELAN experiment] and some updates

rrrokhtar opened the pull request:
### Description
That pull request contains the following mainly:
#### New type of results' exporting at elan experiment 
It is simply a **ZIP** file contains the grouped annotations of **all files**; the exported results will be a compressed file; **experiment_name.zip** contains all experiment's files; each file is similar to the following.
`**Recalling**`; each grouped annotation file contains the annotations of all users to that file and each of them is represented as a single tier for example, an experiment has two tiers (t1 and t2) and three annotators (a1, a2 and a3) and 3 files (f1, f2 and f3);  

The exported results will be zip file contains:
- f1.eaf
- f2.eaf
- f3.eaf

Each elan file will be containing tiers as the following
- t1 (a1)
- t2 (a1)
- t1 (a2)
- t2 (a2)
- t1 (a3)
- t2 (a3)

#### Demo
![prdemo](https://user-images.githubusercontent.com/39674365/203347065-6c02c8b1-ade9-4220-bbc8-e64c9f167ad4.gif)

#### Some side changes
- README.md updated with the new links of GSOC-22 and changed CSOC-21
- Some changed at the current results' code changed "send_file" to "Response" because it has changed, so to keep old-version compatibility replaced with Response.
Flask : [`Changed in version 2.0: download_name replaces the attachment_filename parameter. If as_attachment=False, it is passed with Content-Disposition: inline instead. `](https://flask.palletsprojects.com/en/2.2.x/api/)
 - Changed an id name to the correct logical name

peteruhrig commented:
This looks good! However, we need the option to include all video snippets in the download, because when users come from a concordance, they don't have the right snippets.

rrrokhtar commented:
Updated:
![image](https://user-images.githubusercontent.com/39674365/204031118-957a9e75-ea75-4791-9953-73f2b4df5fa4.png)
- Added option to include all videos at the downloading results for both types (concordance and manual method)
- Another option having the .eaf files only 

- Sample results of concordance uploaded elan experiment attached videos (extracted) 
![image](https://user-images.githubusercontent.com/39674365/204033429-5072d17f-832a-484e-8b91-4f3804adc134.png)

- Sample results of manual uploaded experiment attached videos (extracted) 
![image](https://user-images.githubusercontent.com/39674365/204033339-443a4122-4a7a-4c46-ab14-badff1e646fc.png)

peteruhrig commented:
I think something is going wrong in case of the NewsScape download. These video files are too small. I think they are not the same length as what was displayed to the annotator. They should be downloaded with the timings from the "Display Time" setting, e.g. with 2 seconds to the left and 3 seconds to the right (or whatever the user specified), and the ELAN files should correspond to these snippets.

rrrokhtar commented:
Actually, they are included but most of the start and end are too short. As you are telling, for example a video's url is 
http://pisa.vrnewsscape.ucla.edu/newsscape_mp4_snippet.cgi?file=688b148a-ae1d-11e6-a983-089e01ba0335&start=1010.96&end=1011.44
It is including the start and end from the concordance. 

Concordance file: [concordance_welcome_to_thinned_to_200_for_rapidannotator.txt](https://github.com/RedHenLab/RapidAnnotator-2.0/files/10140950/concordance_welcome_to_thinned_to_200_for_rapidannotator.txt)

rrrokhtar commented:
Updated/Fixed
- Sample download with display time (before=2, after=2)
![image](https://user-images.githubusercontent.com/39674365/205463537-650d2604-b5dc-4ddc-975f-32f3bc6cc06d.png)
